DependencyInjection/Compiler/RegisterPluginsPass.php

```diff
@@ -24,13 +24,18 @@ class RegisterPluginsPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition('swiftmailer.mailer')) {
+        if (!$container->findDefinition('swiftmailer.mailer') || !$container->getParameter('swiftmailer.mailers')) {
             return;
         }
 
-        $definition = $container->findDefinition('swiftmailer.transport');
-        foreach ($container->findTaggedServiceIds('swiftmailer.plugin') as $id => $args) {
```

Comment:
It wasn't looking like a BC break at the time we did that. @lyrixx have you experienced any breakage?